bioptim/misc/options.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, phase: int = -1, list_index: int = -1, name: str = None, type
             Any extra parameters that did not fall in any category
         """
 
+        self.automatic_multiple_phase = False
+        if phase == -1:
+            self.automatic_multiple_phase = True
```

Comment:
I think in this if, phase should be set to 0 to make sure it applies to the first phase and not the last (since some options rely on this fact)

Reply:
I wanted to let it at -1 since OptionGeneric.__init__ was already at -1 by default. What do you think ?